tools/deploy.sh

```diff
@@ -210,8 +210,22 @@ fi
 
 if [ "${DEPLOY_IRONIC}" == "true" ]; then
     pushd "${SCRIPTDIR}"
+    if [[ "${DEPLOY_KEEPALIVED}" == "true" ]]
```

Comment:
nit: preferred form would be placing then on the same line as the if, see for example L211